Stripe/PKPaymentAuthorizationViewController+Stripe_Blocks.m

```diff
@@ -45,6 +50,31 @@ - (void)paymentAuthorizationViewController:(__unused PKPaymentAuthorizationViewC
     }];
 }
 
+- (void)paymentAuthorizationViewController:(__unused PKPaymentAuthorizationViewController *)controller
+                   didSelectShippingMethod:(PKShippingMethod *)shippingMethod
+                                completion:(void (^)(PKPaymentAuthorizationStatus, NSArray<PKPaymentSummaryItem *> *))completion {
+    self.onShippingMethodSelection(shippingMethod, ^(NSArray<PKPaymentSummaryItem *> *summaryItems) {
+        completion(PKPaymentAuthorizationStatusSuccess, summaryItems);
+    });
+}
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated"
+- (void)paymentAuthorizationViewController:(__unused PKPaymentAuthorizationViewController *)controller
+                  didSelectShippingAddress:(ABRecordRef)address
+                                completion:(void (^)(PKPaymentAuthorizationStatus, NSArray<PKShippingMethod *> *, NSArray<PKPaymentSummaryItem *> *))completion {
+    STPAddress *stpAddress = [[STPAddress alloc] initWithABRecord:address];
+    self.onShippingAddressSelection(stpAddress, ^(NSError *validationError, NSArray<PKShippingMethod *>* shippingMethods, NSArray<PKPaymentSummaryItem*> *summaryItems) {
+        if (validationError != nil) {
```

Comment:
I think it's correct – if there was a validation error, the completion status should be `PKPaymentAuthorizationStatusInvalidShippingPostalAddress`.